linux_os/guide/system/kernel_build_config/kernel_config_security/rule.yml

```diff
@@ -0,0 +1,33 @@
+documentation_complete: true
+
+title: 'Enable different security models'
+
+description: |-
+    This allows you to choose different security modules to be configured into your kernel.
```

Comment:
models or modules?

Reply:
It is confusing, but should be `models`:
https://github.com/torvalds/linux/blob/3e732ebf7316ac83e8562db7e64cc68aec390a18/security/Kconfig#L23